### PR TITLE
Update the displayed tutorial robot

### PR DESCRIPTION
**Pull request by abake48**
### Description

This PR starts the transition to make the displayed robot be the Kinova Gen3 7dof. This coincides with the release of the public ROS 2 driver for the Kinova Gen3 arms.

Due to time constraints, this PR only updates the tutorial section of the docs, excluding the Pick and Place with MoveIt Task Constructor tutorial which will be updated in a follow PR.

To ease the transition from the Panda arm, the Panda tutorial example code has only been renamed, not removed, so users can still reference them if they pull down the repo. 

### Checklist
- [ ] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] While waiting for someone to review your request, please consider reviewing [another open pull request](https://github.com/ros-planning/moveit2/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

File: package.xml [hunk 1]
```diff
@@ -60,6 +60,7 @@
   <exec_depend>ros2_control</exec_depend>
   <exec_depend>rviz2</exec_depend>
   <exec_depend>xacro</exec_depend>
+  <exec_depend>kinova_gen3_7dof_robotiq_2f_85_moveit_config</exec_depend>
```
Comment (henningkayser) [inline package.xml:63]: should we add a config to moveit_resources?

File: doc/tutorials/your_first_project/your_first_project.rst [hunk 2]
```diff
@@ -139,7 +139,7 @@ In place of the comment that says "Next step goes here", add this code:
 
   // Create the MoveIt MoveGroup Interface
   using moveit::planning_interface::MoveGroupInterface;
-  auto move_group_interface = MoveGroupInterface(node, "panda_arm");
+  auto move_group_interface = MoveGroupInterface(node, "manipulator");
```
Comment (henningkayser) [inline doc/tutorials/your_first_project/your_first_project.rst:142]: I like the more generic naming here

File: doc/tutorials/your_first_project/panda_hello_moveit.cpp
Comment (henningkayser) [inline doc/tutorials/your_first_project/panda_hello_moveit.cpp:1]: do we still need this?

**Review by henningkayser — COMMENTED** [review]
First superficial pass through GitHub web UI: looks really good so far.
Only more obvious complaint is that all media files have significantly increased in size. Maybe not a huge deal since they are still relatively small.

I'll test this locally before giving the green check mark.

File: package.xml (hunk 1 above)
Comment (abake48) [reply package.xml:63]: [I tried ](https://github.com/ros-planning/moveit_resources/pull/175)but got push back on this. Didn't want to set the precedence that if we want to modify the tutorial robot, it had to be added to `moveit_resources` which would balloon in size.

File: doc/tutorials/your_first_project/panda_hello_moveit.cpp
Comment (abake48) [reply doc/tutorials/your_first_project/panda_hello_moveit.cpp:1]: They aren't being used anymore, but I wasn't sure if we should keep them in the source while we transition to another robot. I could also provide a link within the doc to mention that the tutorial works with another robot

File: package.xml (hunk 1 above)
Comment (henningkayser) [reply package.xml:63]: I see :/

**Review by tylerjw — APPROVED** [review]
I built locally and tried these tutorials, they seem to work for me.

**Pull request merged**